app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) == 3:
+        command, file_name, path = command.split()
+        if file_name == path:
+            return
```

Comment:
it is better to raise some exception so that it is clear to the user if something goes wrong

Reply:
do it